src/Sdk/DTWebApi/WebApi/RunnerRefreshMessage.cs

```diff
@@ -10,11 +11,40 @@ public sealed class RunnerRefreshMessage
     {
         public static readonly String MessageType = "RunnerRefresh";
 
+        [DataContract]
+        public sealed class BrokerPackageMetadata
```

Comment:
i don't think we normally define a DataContract within another DataContract.

Reply:
fixed